css/impact.css

```diff
@@ -0,0 +1,286 @@
+	margin: 0 auto;
+}
+.icon-child .h5-styles {
+	color: var(--color-white);
+	margin: var(--spacing40) 0 0 0;
+}
+
+/* 
+===================================
+Media Queries (overriding styles for different screen sizes)
+
+Note: Here we are using a mobile-first approach, so we start with the smallest screen size and work our way up.
+Start by writing styles for the mobile version of the design on the lines above.
+Anything that needs to change for larger screens should be added to the media queries below.
+
+Feel free to change the min/max-width values to whatever you need. Use variables.
+===================================
+*/
+
+/* Tablet Only */
+@media screen and (min-width: 768px) and (max-width: 1024px) {
+	/* Add styles that only apply to tablet. These are usually fixes to get resposnivenes right */
+}
+
+/* Tablet and Larger */
+@media screen and (min-width: 768px) {
+	/* Add styles that apply to table and desktop */
+	/* Title/Hero section */
+	.title-img {
+		display: none;
+	}
+	/* adding desktop image */
+	.hero {
+		height: 100vh;
+		max-height: 1000px;
+		background-image: url("/assets/images/pages/our-impact/desktop/vr-ar-d.jpg");
+		background-size: cover;
+		background-position: center;
+		background-repeat: no-repeat;
+	}
+	.title-text {
+		width: 100%;
+		height: 100%;
+		background: linear-gradient(
+			92deg,
+			rgba(9, 9, 9, 0.77) 40.35%,
+			rgba(9, 9, 9, 0.74) 44.51%,
+			rgba(9, 9, 9, 0.71) 48.63%,
+			rgba(9, 9, 9, 0) 83.13%
+		);
+
+		/* aligning the title text div */
+		display: flex;
+		align-items: center;
+	}
+	.title-text div {
+		/* adding width instead of margin for responsiveness */
+		width: 70%;
+		margin: 0 auto;
+
+		display: flex;
+		flex-direction: column;
+	}
+	.title-text p,
+	.title-text .h2-styles {
+		/* adding width to make sure text stays on left side */
+		width: 40%;
+		text-align: left;
+		color: var(--color-white);
+	}
+	.title-text .h2-styles {
+		font-size: var(--font56);
+		line-height: var(--line-height58);
+	}
+
+	
+
+	/* icon section */
+	.icons-parent {
+		flex-direction: row;
+		flex-wrap: wrap;
+		justify-content: center;
+	}
+	.icon-child {
+		min-width: 40%;
+	}
+	.yellow-background {
+		background-size: auto 90%;
+	}
+}
+
+/* Desktop Only */
+@media screen and (min-width: 1024px) {
+	/* Add styles that apply specifically to desktop */
+  
+	/* top and bottom section */
+
+	.tech-around,
+	.responsible-parent {
+		width: var(--desktop-wrapper-width);
+		max-width: var(--wrapper-width-max);
+		margin: var(--spacing100) auto;
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
+		align-items: center;
+		gap: var(--spacing100);
+	}
+	.tech-around > * {
+		width: 50%;
+	}
+	.responsible-parent > * {
+		width: 50%;
+	}
+
+	.red-background {
+		background-color: transparent;
+		position: relative;
+	}
+
+	.tech-around .red-background,
+	.responsible-parent .red-background {
+		max-width: 500px;
+		margin: 0 auto;
+	}
+	/* tech around section */
+	.red-block-one {
```

Comment:
Looks like a great start - I noticing some repeated code.
Can you please find similar css properties between **red-block-one** and **red-block-two** and combine them rather than repeating the css declaration?

I think having a common **red-block** class can hold similar styles, you can used **red-block-one** and **red-block-two** to handle positioning. Let me know if you need clarification!

Reply:
Recoded red block code! -Charli